File: Medium/Java/390. Elimination Game.java
class Solution {
    public int lastRemaining(int n) {
        int last = 1;
        if(n == 1){
            return last;
        }else{
            int temp = lastRemaining(n / 2);
            last = 2 * ((n / 2) - temp + 1);
        }
        return last;
    }
}

File: Medium/JavaScript/390. Elimination Game.js
var lastRemaining = function(n) {
    let last = 1;
    if(n == 1){
        return last;
    }else{
        let temp = lastRemaining(Math.floor(n / 2));
        last = 2 * (Math.floor(n / 2) - temp + 1);
    }
    return last;
};
